rules/linux/process_creation/proc_creation_lnx_cat_sudoers.yml

```diff
@@ -4,25 +4,33 @@ status: test
 description: Detects the execution of a cat /etc/sudoers to list all users that have sudo rights
 references:
     - https://github.com/sleventyeleven/linuxprivchecker/
-author: Florian Roth (Nextron Systems)
+author: Florian Roth (Nextron Systems), CheraghiMilad
 date: 2022-06-20
-modified: 2022-09-15
+modified: 2024-12-10
 tags:
     - attack.reconnaissance
     - attack.t1592.004
 logsource:
     category: process_creation
     product: linux
 detection:
-    selection:
+    selection_1:
+        Image|endswith: '/visudo'
```

Comment:
visudo is the correct way to edit the file.
The purpose of the rule is to detect unusual access.

Reply:
In some cases of adversaries Emulation, i use the visudo for reading the sudoers file, and my team can not detected it, my aim  enrichment the rules  for increase coverage.